build-scripts/generate_man_page.py

```diff
@@ -9,13 +9,14 @@
 import ssg.xml
 import os
 import io
+from jinja2 import Template
```

Comment:
we have wrappers and code for this, I suggest you use the wrappers instead of rolling your own. The advantage is that you will be able to pass all of build_config and other SSG specific things.

Reply:
That sounds awesome!